### PR TITLE
COMMERCE-1551 InventoryEngine tests

### PR DESCRIPTION
**Pull request by luca-pellizzon**

### CODE REVIEW

**Comment by marco-leo** [comment]
@luca-pellizzon please resolve conflicts , and ensure test passes on CI

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

File: commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java [hunk 1]
```diff
@@ -213,6 +236,27 @@ public static CommerceInventoryWarehouse addCommerceInventoryWarehouse(
 				sku, quantity);
 	}
 
+	public static CommerceInventoryWarehouseItem
```
Comment (marco-leo) [inline commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java]: @luca-pellizzon I do prefer to have those methods in proper util class and don't mix it

**Comment by luca-pellizzon** [comment]
ci:test

**Comment by luca-pellizzon** [comment]
ci:test

**Pull request merged**